packages/media-utils/src/utils/validate-mime-type.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { UploadError } from './upload-error';
+
+/**
+ * Verifies if the caller (e.g. a block) supports this mime type.
+ *
+ * @param file         File object.
+ * @param allowedTypes List of allowed mime types.
+ */
+export function validateMimeType( file: File, allowedTypes?: string[] ) {
```

Comment:
Nit: should we rename to something like `isMimeTypeAllowed` or something to also convey its boolean return type?

Reply:
> to also convey its boolean return type?
It doesn't have a return type right now, it's either void or throwing an error.

Are you suggesting to change that?

Reply:
No, you're right. I got confused with the `some` callback below.